src/client/opamAuxCommands.ml

```diff
@@ -490,12 +490,11 @@ let check_and_revert_sandboxing root config =
   match OpamFilter.commands env sdbx_wrappers with
   | [] -> config
   | cmd::_ ->
-    OpamFilename.with_tmp_dir @@ fun tmp ->
-    let test_file = OpamFilename.(to_string Op.(tmp // "check")) in
+    let test_file = "$TMPDIR/opam-sandbox-check-out" in
```

Comment:
The sandbox script is expected to set it; but indeed that should be documented...

Reply:
This is in order to get canonical output, right? Rather than altering the command, wouldn't it be better to alter `OpamProcess.verbose_print_cmd` to detect the string and do the conversion instead?

Reply:
We did something similar with negative debug levels stripping out the timestamps...